apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+
+It is recommended to use the empty site as a starting point when you are creating your own websites outside the scope of this blog. The basic site is more for quick demo purposes, and would require excess cleanup when used for a client project.  However, for this blog, you will select to use the basic site.   
+![Choose Site Template](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1480ca22c0724a7496c9617b5f65cd6a?v=cf3753f9) 
+
+3. Name the site `Company DEV`.  
+![Name new Site and Site Collection](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1dc9460c15fd452681187485fae6fe37?v=8feccf45) 
+
+4. Select a pre-configured language, in this case, English.  
+
+At this time, you can also select an existing Site identifier, or create a new one. This is required when connecting CDP for embedded analytics and personalization.  
+
+Now, let’s start the creation process of the site.  
+
+5. Click `create website`, which will scaffold the website in the background. This will take a few minutes.  
+
+Once your site is scaffolded in the background, you can access it using XM Cloud Pages and start editing the content. 
+
+![Open Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/6438c1971da24b8f8564e3e48335d985?v=c5f57f17) 
+ 
+6. Click on the website, which opens up a WSYWIG editor where you can begin to edit the content, see components that you can use to edit the site, and switch between languages and your sites (top nav bar).   
+![In Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/9fb983d0eeec435797a1115f4ae801df?v=2cce57d4) 
+ 
+
+## Working locally  
+
+Let’s now navigate to your GitHub account and download the repository that was cloned earlier by the Sitecore XM Cloud Deploy App.  
+![GitHub - Clone Repository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/239fa0b1ce314ca088bc830ee7a699e8?v=ba0d1b2e) 
+
+### Download Repository  
+
+1. In your repositories overview, find the newly created xm-cloud-tutorial-series repo.  
+2. Download the repository to your local file system using the GitHub CLI.Now, open the newly created folder in VS Code, or your text editor of choice.  
+3. As per the README, run the init script that was delivered with your source code repository passing the path to your XM Cloud license file, and the admin password of your XM Cloud instance (“b”, in this case). The init script initializes the .env file with everything necessary.  
+![In Command Line Tool - Run init script](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8cb688885bc74cf9a3b44468dd58ff57?v=0ca77d8f) 
+ 
+ 
+*A note on passwords:  
+You can use any password and will of course have to use a secure password when dealing with client projects.  But for now we use “b”, for tradition’s sake. * 
+
+### Connect the App to the XM Cloud Instance  
+
+A few things need to be configured upfront to connect to the GraphQL endpoint.  
```

Comment:
I'd re-iterate what we're doing with this GraphQL endpoint as we re-introduce it, as its been a while since it was mentioned. So, I would say: 

'As you may remember from the introduction, you need to use a GraphQL endpoint to query the content coming from XM Cloud.  A few things need to be configured upfront...'

Or something like that.

Reply:
Resolved.